src/olympia/abuse/urls.py

```diff
@@ -1,15 +1,15 @@
-from django.conf.urls import include, url
+from django.conf.urls import include
+from django.urls import re_path
 
 from rest_framework.routers import SimpleRouter
 from .views import AddonAbuseViewSet, UserAbuseViewSet
 
-
```

Comment:
These extra lines aren't enforced by a linting rule but we've decided we like them for consistency (so the imports are always separated by 2 lines from the non-imports). So can you put back all the extra lines at the top in the urls.py files.

Reply:
I think it's good to go after that.

Reply:
Changes are done with line spacing of 2 between imports and non-imports.